### PR TITLE
[zh] Sync Announcing the alpha availability of WebAssembly Plugins blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate Announcing the alpha availability of WebAssembly Plugins:
```
content/zh/blog/2021/wasm-api-alpha/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2021/wasm-api-alpha/index.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+title: WebAssembly 插件 Alpha 版可用性公告
+description: 介绍新的 Wasm 插件 API 以及 Envoy 和 Istio 中基于 Wasm 的插件支持的更新。
+publishdate: 2021-12-16
+attribution: "Daniel Grimm (Red Hat), Pengyuan Bian (Google), Takeshi Yoneda (Tetrate); Translated By: Wilson Wu (DaoCloud)"
```
Comment (windsonsea) [inline content/zh/blog/2021/wasm-api-alpha/index.md:5]:
```suggestion
attribution: "Daniel Grimm (Red Hat), Pengyuan Bian (Google), Takeshi Yoneda (Tetrate); Translated by Wilson Wu (DaoCloud)"
```
Not sure whether this form is agreed mostly, which can have some more variants:
- Wilson Wu (DaoCloud)
- Wilson Wu (DaoCloud) translated/localized
- localized by Wilson Wu (DaoCloud)

File: content/zh/blog/2021/wasm-api-alpha/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+可扩展性推进到稳定、可靠且易于使用的程度，我们很高兴地宣布
+Istio 1.12 中对 Wasm 可扩展性提供 Alpha 支持！在接下来的内容中，
+我们将逐步介绍 1.12 版中针对 Wasm 支持所做的更新。
+
+## 新的 WasmPlugin API {#new-wasmplugin-api}
+
+通过 `extensions.istio.io` 命名空间中的新 `WasmPlugin` CRD，
+我们引入了一个用于自定义 Wasm 模块对 Istio 代理功能进行扩展的新顶层 API。
+这项出色的工作归功于过去两年 [Proxy-Wasm](https://github.com/proxy-wasm)
+社区出色的规范和实施基础上。从现在开始，您不再需要使用 `EnvoyFilter`
```
Comment (windsonsea) [inline content/zh/blog/2021/wasm-api-alpha/index.md:23]:
```suggestion
社区出色的规范和实施基础。从现在开始，您不再需要使用 `EnvoyFilter`
```

File: content/zh/blog/2021/wasm-api-alpha/index.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+`pluginConfig` 字段用于 Wasm 插件的具体配置。
+在此字段中输入的任何内容都将通过 JSON 格式进行编码并传递到您的过滤器中，
+您可以在 Proxy-Wasm SDK 的配置回调中访问它。例如，您可以使用
+[C++ SDK](https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/fd0be8405db25de0264bdb78fae3a82668c03782/proxy_wasm_api.h#L329-L331)
+中的 `onConfigure`、[Rust SDK](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/blob/v0.1.4/src/dispatcher.rs#L255)
+中的 `on_configure` 或 [Go SDK](https://github.com/tetratelabs/proxy-wasm-go-sdk/blob/v0.15.0/proxywasm/types/context.go#L74)
+中的 `OnPluginStart` 回调对这些配置进行检索。
+
+`url` 字段指定了 Wasm 模块的获取位置。请注意本示例中的 `url`
```
Comment (windsonsea) [inline content/zh/blog/2021/wasm-api-alpha/index.md:70]:
```suggestion
`url` 字段指定了 Wasm 模块的拉取位置。请注意本示例中的 `url`
```

**Review by windsonsea — COMMENTED** [review]
Newline can be a range around col 80 - 120. The current layout is relatively narrow and hurry to go to the next line.

File: content/zh/blog/2021/wasm-api-alpha/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/blog/2021/wasm-api-alpha/index.md:5]: For the clear format, I think Wilson Wu (DaoCloud) translated is a good way, still need more suggestions @ericvn @hanxiaop @kebe7jun

Comment (kebe7jun) [reply content/zh/blog/2021/wasm-api-alpha/index.md:5]: `Translated by Wilson Wu (DaoCloud)` seems clear and unify.

Comment (hanxiaop) [reply content/zh/blog/2021/wasm-api-alpha/index.md:5]:
```suggestion
attribution: "Daniel Grimm (Red Hat), Pengyuan Bian (Google), Takeshi Yoneda (Tetrate); translated by Wilson Wu (DaoCloud)"
```
Same as @kebe7jun, however not sure if we need to capitalize these words after `;`

Comment (windsonsea) [reply content/zh/blog/2021/wasm-api-alpha/index.md:5]: chatGPT said both are correct 😄 :

```
attribution: "..., Takeshi Yoneda (Tetrate); translated by Wilson Wu (DaoCloud)"
attribution: "..., Takeshi Yoneda (Tetrate); Translated by Wilson Wu (DaoCloud)"
```

Comment (wilsonwu) [reply content/zh/blog/2021/wasm-api-alpha/index.md:5]: So I will accept this: Translated by Wilson Wu (DaoCloud)

**Review by windsonsea — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**